src/Data/Functor/Linear/Internal/Traversable/Generic.hs

```diff
@@ -84,6 +90,19 @@ instance GTraversable UWord where
 
 -- | Implementation of 'Data.Functor.Linear.traverse' for types which derive
 -- (linear) 'Generics.Linear.Generic1'.
+--
+-- Use as
+--
+-- > data T
+-- > $(deriveGeneric1 ''Pair)
+-- >
+-- > instance Traversable T where
+-- >   traverse = genericTraverse
+--
+-- Note that, contrary to many other classes in linear-base, we can't define
+-- `Traversable T` using deriving via, because the
+-- [role](https://downloads.haskell.org/ghc/latest/docs/html/users_guide/exts/roles.html)
```

Comment:
The link for `role` may be hard to hit on mobile browsers and other touchscreens, since it's so short. How much does that matter?

Reply:
I think it matters a little, but not a ton (you can always zoom on mobile). So if you have a good solution, I'll take it. But let's not spend a lot of time on it.